package.json

```diff
@@ -12,7 +12,7 @@
     "docz:dev": "docz dev",
     "docz:build": "docz build",
     "lint": "tslint --project tsconfig.json {src}/**/*.ts && prettier-check --ignore-path .gitignore src/**/*.ts",
-    "postinstall": "lightercollective postinstall",
+    "prepare": "lightercollective postinstall",
```

Comment:
Newer npm (& Yarn) versions include support for the prepare script that is run after each install run but only in development mode.